portal-api/src/v1/gef/controllers/validation/exporter.js

```diff
@@ -70,13 +68,8 @@ const exporterCheckEnums = (doc) => {
   return enumErrors.length === 0 ? null : enumErrors;
 };
 
-const exporterValidation = (doc) => ({
-  required: hasRequiredItems(doc),
-});
-
```

Comment:
I think this is incorrect and was fixed in other PR that's been merged, do a pull?

Reply:
It's in a different PR, yes. 
I don't think the function is being used though as I couldn't find a reference to it anywhere else

Reply:
You're right! Sorry, I confused myself.